Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=59036e94732edc2fb957465008c68bbcfc6736fa
Commit:     59036e94732edc2fb957465008c68bbcfc6736fa
Parent:     1902869019918411c148c18cc3a22aade569ac9a
Author:     Andy Shevchenko <andy@smile.org.ua>
AuthorDate: Mon Feb 4 23:57:49 2008 -0800
Committer:  Greg Kroah-Hartman <gregkh@suse.de>
CommitDate: Thu Feb 21 15:38:48 2008 -0800

    USB: usb: yet another Dell wireless CDMA/EVDO modem
    
    Add native support of the Dell wireless CDMA/EVDO modem.
    
    # modprobe usbserial vendor=0x413c product=0x8129
    
    Following seesion lines describe modem itself
    at
    OK
    ati3
    Manufacturer: NOVATEL WIRELESS INCORPORATED
    Model: EXPEDITE ET620
    Revision: M6500C-BBIRD_TLS_MINI_DELL-Q40306.166 [Aug 25 2006 14:00:00]
    ESN: 0x5B39071D
    +GCAP: +CIS707-A, CIS-856, +MS, +ES, +DS, +FCLASS
    
    OK
    
    Signed-off-by: Andy Shevchenko <andy@smile.org.ua>
    Cc: Matthias Urlichs <smurf@smurf.noris.de>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/usb/serial/option.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/usb/serial/option.c b/drivers/usb/serial/option.c
index a812698..e3e71c5 100644
--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -181,6 +181,7 @@ static struct usb_device_id option_ids[] = {
 	{ USB_DEVICE(DELL_VENDOR_ID, 0x8118) },	/* Dell Wireless 5510 Mobile Broadband HSDPA ExpressCard == Novatel Merlin XU870 HSDPA/3G */
 	{ USB_DEVICE(DELL_VENDOR_ID, 0x8128) },	/* Dell Wireless 5700 Mobile Broadband CDMA/EVDO Mini-Card == Novatel Expedite E720 CDMA/EV-DO */
 	{ USB_DEVICE(DELL_VENDOR_ID, 0x8136) },	/* Dell Wireless HSDPA 5520 == Novatel Expedite EU860D */
+	{ USB_DEVICE(DELL_VENDOR_ID, 0x8129) },	/* Dell Wireless 5700 Mobile Broadband CDMA/EVDO Mini-Card == Novatel Expedite ET620 CDMA/EV-DO */
 	{ USB_DEVICE(DELL_VENDOR_ID, 0x8137) },	/* Dell Wireless HSDPA 5520 */
 	{ USB_DEVICE(ANYDATA_VENDOR_ID, ANYDATA_PRODUCT_ADU_E100A) },
 	{ USB_DEVICE(ANYDATA_VENDOR_ID, ANYDATA_PRODUCT_ADU_500A) },
